pkg/messagebroker/kafka.go

```diff
@@ -571,3 +572,66 @@ func (k *KafkaBroker) Close(ctx context.Context) error {
 
 	return nil
 }
+
+// AddTopicPartitions adds partitions to an existing topic
```

Comment:
add comments here, this could be breaking if the topic is being used for ordered messages, should be used with caution :)